### ISSUE TITLE
Check spelling in the commit message

### ISSUE THREAD

**Issue by sparrowek** [streetsidesoftware/vscode-spell-checker#1463]
Is it possible to add such a feature or is it blocked by https://github.com/Microsoft/vscode/issues/35571 ?

**Comment by felipecock** [comment]
I wish so much. Many people are waiting for it for years.

**Comment by globalhuman** [comment]
Now that the above is unblocked is this now possible?

**Comment by Jason3S** [comment]
@globalhuman,

This should be working.

![image](https://user-images.githubusercontent.com/3740137/196869544-d6c69109-e22c-4e49-ba04-ec3abf4f541c.png)


It is possible you might need to adjust the schemas to include `vscode-scm`:

![image](https://user-images.githubusercontent.com/3740137/196869452-add40231-133d-4f70-9c55-bd0e7eb8d7bd.png)

**Comment by Jason3S** [comment]
I'm going to close this as completed. Please open a new issue if it is not working as expected.
